### ISSUE TITLE
CGI security fixes for old versions are not in repository

### ISSUE THREAD

**Issue by headius** [ruby/cgi#31]
We were informed today of the new CVE-2021-33621 and want to update the copy of cgi shipped with JRuby 9.3. Unfortunately Ruby 2.6 has not been patched and I cannot find the related branches for older versions of CGI anywhere.

What patch went into 0.1.0.2 and friends? Where are the branches for those release lines?

**Comment by headius** [comment]
ping @hsbt

**Comment by headius** [comment]
I also noticed that the 0.1.0.2 tag points at a revision where there's a VERSION of 0.1.0.1. Is the fix actually in this release at all?

**Comment by hsbt** [comment]
>Is the fix actually in this release at all?

Yes. see https://github.com/ruby/cgi/compare/v0.1.0.1...v0.1.0.2

**Comment by headius** [comment]
Aha I see the version.rb was added and the VERSION moved out of cgi.rb.

Where does the branch for this exist? I am very confused how this commit is in the repository when there's only master and master HEAD is 0.3.5.

**Comment by headius** [comment]
Note that without being able to reach the tag from a branch, none of the security tags show up in https://github.com/ruby/cgi/tags.

**Comment by hsbt** [comment]
It's good to manage with branch strategy. I think too. But the current cgi gem maintained only each stable CRuby releases. So there are no additional commits on this repository.

If We need to push the additional feature without CRuby releases, I will create the stable branches like `v0.2`. Is it ok?

**Comment by headius** [comment]
I think it would be good and safe to push new branches for the recently patched releases, so those tags can show up on GitHub and we can easily see those versions' histories.

**Comment by hsbt** [comment]
All affected versions (cgi 0.1.0.x for Ruby 2.6/2.7) are now EOL, and I won't be cutting maintenance branches for them.

The security commits remain preserved via their tags, so closing this.
